tests/locationConfigTests.json

```diff
@@ -85,5 +85,16 @@
           "azureBlobSAS": "sv=2015-04-05&sr=b&si=tutorial-policy-635959936345100803&sig=9aCzs76n0E7y5BpEi2GvsSv433BZa22leDOZXX%2BXXIU%3D",
           "azureContainerName": "s3test"
         }
+    },
+    "gcp-test": {
+        "type": "gcp",
+        "legacyAwsBehavior": true,
+        "details": {
+            "gcpEndpoint": "https://www.googleapis.com/storage/v1",
+            "bucketMatch": true,
+            "projectId": "zenko-hackathon",
+            "gcpBucketName": "my-zenko-bucket",
+            "credentialsEnv": "GCP_CRED"
```

Comment:
The credentialsEnv stores the path to the keyfile and there is no default path, so we stored the path in an environment variable.

Reply:
I'll have to think about how to configure this. If you put together some simple instructions on how your approach works -- what each of these items are and what you need to do to run this (set the env variable, etc.) -- it would be helpful. Is there any option to just put the keys here?

Reply:
@tomateos, did you look at google's interoperability api at all? https://cloud.google.com/storage/docs/interoperability
It makes google much closer to s3 calls meaning we could use a very similar configuration format as we do for s3 buckets and maybe even reuse the aws s3 client code. I played with it a little yesterday and it does puts/gets just like you are hitting an s3 endpoint. It does not work with multipart-uploads so we will have to think about how to handle that. Thoughts?